### DISCUSSION TITLE
Deselecting Rainbowify JSX(HTML inside JS) in settings does nothing

### DISCUSSION THREAD

**Discussion by j-d-ha** [izhangzhihao/intellij-rainbow-brackets#2764] [Q&A] [answered]
### Have you checked the issues and discussions to ensure there are no duplicates?

Yes

### Your programming languages

javascript

### Free or paid?

Free users

### Expected Behavior

When I deselect `Rainbowify JSX(HTML inside JS)` in settings, HTML in the .jsx (or .tsx) file should no longer be rainbowified. 

### Current Behavior

There is no change. HTML in the file is still rainbowified. 

![image](https://github.com/user-attachments/assets/da686f1a-a1b2-488e-b43a-6faabb77ef4f)

### Code snippet for reproduce

```
import { StrictMode } from 'react'
import { createRoot } from 'react-dom/client'
import App from './App.tsx'
import './index.css'

createRoot(document.getElementById('root')!).render(
  <StrictMode>
    <App />
  </StrictMode>,
)
```

### Your Environment

WebStorm 2024.1.5
Build #WS-241.18034.50, built on June 19, 2024
Runtime version: 17.0.11+1-b1207.24 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.

**Comment by izhangzhihao** [comment, marked as answer]
I hope this answers your question, this issue will be converted to a discussion, if you have more questions pls leave comments in the new discussion
<img width="913" alt="image" src="https://github.com/user-attachments/assets/ccac878b-46b2-4002-85c7-38f3b406e900">
